docs/api.yml

```diff
@@ -566,9 +598,15 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/NoteLicenseRef'
-        refereed:
-          type: string
-          example: true
+        free2read:
+          type: object
+          properties:
+            start:
+              type: string
+              format: date
+            end:
+              type: string
+              format: date
```

Comment:
Can this have a `description` that makes a note that this value if present should be ignored and may be deprecated in future.